otlp/src/otlp/logs/component.cpp

```diff
@@ -38,6 +38,7 @@ LoggerComponent::LoggerComponent(const components::ComponentConfig& config,
       "otlp-tracer", endpoint);
 
   LoggerConfig logger_config;
+  bool send_logs = config["send-logs"].As<bool>(true);
```

Comment:
maybe smth like that
```
sinks:
  logs: both
  tracing: otlp
```

Reply:
Added optional sinks block (by default logs and traces are to be sent via otlp).